src/libsystemd/libsystemd.sym

```diff
@@ -573,4 +573,6 @@ global:
 
 LIBSYSTEMD_240 {
         sd_bus_message_readv;
+        sd_bus_set_default_method_call_timeout;
+        sd_bus_get_default_method_call_timeout;
```

Comment:
I think this should be called just: `sd_bus_set_method_call_timeout` or so. It's a pretty long function name even then, and one could argue that `BUS_DEFAULT_TIMEOUT` is the actual default, and this new call would already have an effect to change things from the default and hence shouldn't be called default. hope that logic makes some sense.